xds/internal/balancer/clusterresolver/e2e_test/eds_impl_test.go

```diff
@@ -367,10 +380,11 @@ func (s) TestEDS_MultipleLocalities(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	// Change the weight of one locality and ensure weighted roundrobin.  Since
-	// RPCs are weighted roundrobined across localities, and locality2 has two
-	// times the weight of locality1, locality2's backend will receive four
-	// times the traffic of locality1's backend.
+	// Change the weight of locality2 and ensure weighted roundrobin.  Since
+	// locality2 has twice the weight of locality3, it will be picked twice as
+	// frequently as locality3 for RPCs. And since locality2 has half the number
+	// of backends as locality3, its backends will receive four times the
+	// traffic of locality3's backend.
```

Comment:
Sorry, I honestly don't care that much, but this is still logically incorrect. Four times the traffic of *either* one of locality3's backend. Or switch to two times the traffic of locality3.

Reply:
Done.